app/services/create_reservation.rb

```
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class CreateReservation
+  def call
+    return false unless seats_selected?
+
+    ActiveRecord::Base.transaction do
+      reservation.save!
+      create_tickets
+    rescue StandardError
```

Why do you suppress this exception here?